### PR TITLE
Fix trivial megacheck issues

### PR DESCRIPTION
**Pull request by mjtrangoni**
Hi @bergquist @torkelo,

related to #10381, I fixed some trivial *megacheck* issues.

### CODE REVIEW

File: pkg/services/sqlstore/user_auth_test.go [hunk 1]
```diff
@@ -16,7 +16,7 @@ func TestUserAuth(t *testing.T) {
 	Convey("Given 5 users", t, func() {
 		var err error
 		var cmd *m.CreateUserCommand
-		users := []m.User{}
+		// users := []m.User{}
```
Comment (torkelo) [inline pkg/services/sqlstore/user_auth_test.go]: could you describe the fix here, unused variable?

Comment (mjtrangoni) [reply pkg/services/sqlstore/user_auth_test.go]: Yes, should I remove it?

Comment (torkelo) [reply pkg/services/sqlstore/user_auth_test.go]: yes

**Comment by mjtrangoni** [comment]
@torkelo fixed.

**Comment by torkelo** [comment]
thanks!

**Pull request merged**